CMakeLists.txt

```diff
@@ -44,7 +44,6 @@ include_directories(SYSTEM
   ${Boost_INCLUDE_DIRS}
   ${DART_INCLUDE_DIRS}
   ${PYTHON_INCLUDE_DIRS}
-  "/opt/ros/indigo/include" # TODO: This should not be necessary.
```

Comment:
Does anyone remember why we had this in the first place?

Reply:
Hm, no idea.

> Are there any components of DART that link against ROS right now that we would need to create bindings to?

DART doesn't have any component depends on ROS.

Reply:
Ok, then I guess we should probably remove it :woman_shrugging: